### ISSUE TITLE
Send Non-Prefect Logs to Prefect Cloud

### ISSUE THREAD

**Issue by braunreyes** [PrefectHQ/prefect#1998]
## Current behavior
We have an internal library that we use to abstract away some things.  This library has a logger that gets created for use within the library's methods.  These logs are defaulted with a `NullHandler()`.  We have a method `get_stdout_logger` that will setup a `StreamHandler` for stdout for these logs.  We cannot get these logs to show up in Prefect Cloud.  We have tried adding the `CloudHandler` to our library logger, but still not working.

## Proposed behavior
We would like some way to get non-Prefect logs into Prefect Cloud.

## Example
example of how we are trying to do this.  This done before defining tasks and flow objects in our flow file.
```
from ccde.util_logging import get_stdout_logger
from prefect import task, Flow, utilities

LOGGER = utilities.logging.get_logger('prefect')

cloud_handler = utilities.logging.CloudHandler()
cloud_handler.setLevel('INFO')
get_stdout_logger('ccde').addHandler(cloud_handler)
```
get_stdout_logger
```
# function to return logger for application
def get_stdout_logger(name, propagate=None):
    # initialize logger
    logger = logging.getLogger(name)
    # get log level from config
    log_level = 'INFO'
    # set log level
    logger.setLevel(log_level)
    # create console log handler
    log_stream = logging.StreamHandler(sys.stdout)
    # set format
    log_stream.setFormatter(LOG_FORMAT)
    # add handler
    logger.addHandler(log_stream)
    logger.propagate = True
    return logger
```

**Comment by braunreyes** [comment]
#2010 
This PR is mean to add this

**Comment by joshmeek** [comment]
Closed in #2010
